.github/CONTRIBUTING.md

```diff
@@ -140,7 +140,7 @@ Then, we can create the databases (if they don't already exist):
 
 ```
 $ sudo -u postgres createuser you_username --createdb
```

Comment:
@Chocobozzz at first I ran this command, thats the reason the user already existed. And here is no `--superuser`.
So theoretically, here `sudo` would be needed, right?